dts/bindings/sensor/sensor-device.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sensor Device
+
+include: base.yaml
+
+properties:
+  vendor:
+    type: string
+    required: false
+
+  model:
+    type: string
+    required: false
+
+  friendly-name:
+    type: string
+    required: false
+
+child-binding:
```

Comment:
Is this really common?  Feels specific to a given type of sensor?

Reply:
I included this to address @yuval's concern in #48148:

> `vendor`, `model`, and `friendly-name` should not be required at the individual node level, it's vulnerable to having a mismatch. Since there will be one node per sensor type here, several nodes may point to a single physical sensor.

Some sensor devices support multiple types of channels - e.g., a 9-axis IMU that supports 3d accel, 3d mag, and 3d gyro channels. My current thinking is that we will need a common set of base properties, like units, full scale range and sample rate, to describe each group of channels as a subnode of the sensor device.

I think we can pull the child-binding out for now and iterate on it in follow up PRs.

Reply:
I'm not disagreeing with your analysis, @MaureenHelm , but I am wondering whether you've looked at [iio bindings](https://github.com/torvalds/linux/tree/master/Documentation/devicetree/bindings/iio) when doing it. It seems to me like iio is the de facto standard for sensor bindings in Linux and I want to understand how decisions to deviate from its bindings, if any, were made, especially given @galak 's tepid enthusiasm for [deviating from Linux](https://github.com/zephyrproject-rtos/zephyr/pull/49287) (conveyed to me privately)